src/App.jsx

```diff
@@ -0,0 +1,114 @@
+import { setWidget, WidgetDataType, WidgetsProvider } from '@sitecore-discover/react';
+import { createTheme } from '@sitecore-discover/ui';
+import 'animate.css/animate.min.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import PropTypes from 'prop-types';
+import React, { lazy, Suspense, useEffect } from 'react';
+import { BrowserRouter, Navigate, Route, Routes, useLocation } from 'react-router-dom';
+import Footer from './components/Footer';
+import Header from './components/Header';
+import TopMenu from './components/TopMenu';
+import { BASE_PATH } from './helpers/constants';
+import { CartProvider } from './hooks/cart';
+import { Recommendation } from './widgets/BasicRecommendation';
+// eslint-disable-next-line import/extensions
+import 'bootstrap/dist/js/bootstrap.bundle.js';
+import 'react-notifications-component/dist/theme.css';
+import './App.css';
+import { WishlistProvider } from './hooks/wishlist';
+
+const HomeView = lazy(() => import('./views/Home'));
+const ProductDetailView = lazy(() => import('./views/Product/Detail'));
+const CartView = lazy(() => import('./views/Cart/Cart'));
+const OrderView = lazy(() => import('./views/Order'));
+const NotFoundView = lazy(() => import('./views/pages/404'));
+const InternalServerErrorView = lazy(() => import('./views/pages/500'));
+const SearchView = lazy(() => import('./views/SearchView'));
+const CategoryView = lazy(() => import('./views/CategoryView'));
+const WishlistView = lazy(() => import('./views/Wishlist'));
+
+const ScrollToTop = ({ children }) => {
+  const { pathname } = useLocation();
+
+  useEffect(() => {
+    window.scrollTo({
+      top: 0,
+      left: 0,
+      behavior: 'smooth',
+    });
+  }, [pathname]);
+
+  return children;
+};
+
+ScrollToTop.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+function App() {
+  const { style, setStyle } = createTheme({
+    typography: {
+      fontFamilySystem: '"Source Sans Pro", sans-serif',
+    },
+    palette: {
+      primary: {
+        main: '#43474c',
+        light: '#f1f2f3',
+        dark: '#0d0d0d',
+      },
+    },
+  });
+  const bodyElement = document.body;
+  setStyle(bodyElement, style);
+
+  setWidget('hs_random', {
+    type: WidgetDataType.RECOMMENDATION,
+    component: Recommendation,
+    options: {
+      properties: {
+        initial: {
+          title: 'Similar Items',
+        },
+      },
+    },
+  });
```

Comment:
Can we move these line outside the component, please 🤓?

Reply:
Good catch! It was old, it is not needed any more